dask_ml/model_selection/_incremental.py

```diff
@@ -515,21 +515,25 @@ def __init__(
         self.prefix = prefix
         super(BaseIncrementalSearchCV, self).__init__(estimator, scoring=scoring)
 
-    def _validate_parameters(self, X, y):
+    async def _validate_parameters(self, X, y):
         if (self.max_iter is not None) and self.max_iter < 1:
             raise ValueError(
                 "Received max_iter={}. max_iter < 1 is not supported".format(
                     self.max_iter
                 )
             )
 
-        # Make sure dask arrays are passed so error on unknown chunk size is raised
         kwargs = dict(accept_unknown_chunks=True, accept_dask_dataframe=True)
         if not isinstance(X, dd.DataFrame):
             X = self._check_array(X, **kwargs)
-        if not isinstance(y, dd.Series):
+        if not isinstance(y, dd.DataFrame):
             y = self._check_array(y, ensure_2d=False, **kwargs)
```

Comment:
Why this change?  I would expect that `y` would more often be a Series than a DataFrame.  Should we include either type?

Reply:
Thanks for the catch, I think this is a bad merge. I've fixed it.